src/arrayMethodSort.js

```diff
@@ -4,8 +4,36 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    if (compareFunction === undefined) {
+      for (let i = 0; i < this.length; i++) {
+        for (let k = 0; k < this.length; k++) {
+          if ('' + this[k] > '' + this[k + 1]) {
```

Comment:
The string conversion here is not necessary for comparison. JavaScript will coerce to strings if needed when comparing with '>'.